flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/ConnectedStreams.java

```diff
@@ -234,62 +235,62 @@ public StreamExecutionEnvironment getExecutionEnvironment() {
 	}
 
 	/**
-	 * Applies the given {@link TimelyCoFlatMapFunction} on the connected input streams,
+	 * Applies the given {@link CoProcessFunction} on the connected input streams,
 	 * thereby creating a transformed output stream.
 	 *
 	 * <p>The function will be called for every element in the streams and can produce
 	 * zero or more output. The function can also query the time and set timers. When
 	 * reacting to the firing of set timers the function can emit yet more elements.
 	 *
```

Comment:
I would transform it into (to also show the distinction between flatMap): 

<p>The function will be called for every element in the input streams and can produce zero or more
	 * output elements. Contrary to the {@link #flatMap(CoFlatMapFunction)} function, this function can also
	 * query the time and set timers. When reacting to the firing of set timers the function can directly
	 * emit elements and/or register yet more timers.